assets/app/view/game/corporation.rb

```diff
@@ -638,9 +638,10 @@ def render_loans
           loan_props[:style][:color] = contrast_on(color)
         end
 
+        corp_loans_text = @corporation.respond_to?(:corp_loans_text) ? @corporation.corp_loans_text : 'Loans'
         [
           h('tr.ipo', loan_props, [
-            h('td.right', 'Loans'),
+            h('td.right', corp_loans_text),
```

Comment:
Only used once, so you don't need a local var. You can just put the logic here. Local vars are useful if used more than once.